app/controllers/FileUploadController.scala

```diff
@@ -94,6 +73,80 @@ class FileUploadController @Inject()(fileUploadConnector: FileUploadConnector,
       }
   }
 
+  def post: Action[AnyContent] = Action.async { implicit request =>
+    isAuthorised.flatMap {
+      case Right(userId) =>
+        getFile(request) match {
+          case file if (file.filename.length == 0) =>
```

Comment:
.length == 0 can be replaced with isEmpty. In the background it does the same thing.